src/main/java/com/puppycrawl/tools/checkstyle/checks/header/AbstractHeaderCheck.java

```diff
@@ -77,19 +77,21 @@ protected List<String> getHeaderLines() {
     }
 
     /**
-     * Setter to specify the charset to use when reading the headerFile.
+     * Setter to specify the character encoding to use when reading the headerFile.
      *
      * @param charset the charset name to use for loading the header from a file
+     * @since 4.4
```

Comment:
Also need commits showing this version number is correct.

Reply:
all since are reverted. map is updated with values from xdoc